app/views/shared/newrelic/_browser_instrumentation.html.erb

```diff
@@ -1,4 +1,6 @@
-;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"<%= IdentityConfig.store.newrelic_browser_key %>",applicationID:"<%= IdentityConfig.store.newrelic_browser_app_id %>",sa:1}
+  ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
+  ;NREUM.loader_config={accountID:"<%= IdentityConfig.store.newrelic_account_id %>",trustKey:"<%= IdentityConfig.store.newrelic_browser_trust_key %>",agentID:"<%= IdentityConfig.store.newrelic_browser_agent_id %>",licenseKey:"<%= IdentityConfig.store.newrelic_browser_license_key %>",applicationID:"<%= IdentityConfig.store.newrelic_browser_app_id %>"}
+  ;NREUM.info={beacon:"gov-bam.nr-data.net",errorBeacon:"gov-bam.nr-data.net",licenseKey:"<%= IdentityConfig.store.newrelic_browser_license_key %>",applicationID:"<%= IdentityConfig.store.newrelic_browser_app_id %>",sa:1}
```

Comment:
I know it would add a few extra `"` because they'd be string keys, but what about building this as a hash and calling `.to_json` on it? Or potentally calling `j` or `escape_javascript` on these?
```suggestion
  ;NREUM.info=<%= {beacon:"gov-bam.nr-data.net",errorBeacon:"gov-bam.nr-data.net",licenseKey: IdentityConfig.store.newrelic_browser_license_key, applicationID: IdentityConfig.store.newrelic_browser_app_id,sa:1}.to_json %>
```

Reply:
> I know it would add a few extra `"` because they'd be string keys, but what about building this as a hash and calling `.to_json` on it? Or potentally calling `j` or `escape_javascript` on these?

Yeah, makes sense. It required some unescaping (`<%==` instead of `<%=`) to render as `"` instead of `&quot;`, but otherwise implemented as part of c31c24aea.